### PR TITLE
W-11308645: After calling an external SOAP endpoint, Error Objects are generated differently in MRT 4.3.0 and MRT 4.4.0 

### PR DESCRIPTION
**Pull request by IvanAndresFritzler**

### CODE REVIEW

File: modules/core-components/src/main/java/org/mule/runtime/core/internal/routing/UntilSuccessful.java [hunk 1]
```diff
@@ -108,7 +114,7 @@ public CoreEvent process(CoreEvent event) throws MuleException {
   @Override
   public Publisher<CoreEvent> apply(Publisher<CoreEvent> publisher) {
     return new UntilSuccessfulRouter(this, publisher, nestedChain, processingStrategy, expressionManager, shouldRetry, timer,
-                                     maxRetries, millisBetweenRetries)
+                                     maxRetries, millisBetweenRetries, featureFlaggingService.isEnabled(SUPPRESS_ERRORS))
```
Comment (elrodro83) [inline modules/core-components/src/main/java/org/mule/runtime/core/internal/routing/UntilSuccessful.java:117]: obtain the value during initialization

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Comment by IvanAndresFritzler** [comment]
--validate

**Comment by IvanAndresFritzler** [comment]
--validate

**Review by elrodro83 — APPROVED** [review]

**Comment by IvanAndresFritzler** [comment]
--validate

**Pull request merged**